provisioning/src/index.js

```diff
@@ -443,36 +437,12 @@ async function testWorkflowitemUpdate(folder) {
       !Array.isArray(itemWithDocuments.data.documents) ||
       itemWithDocuments.data.documents.length !== 2
     ) {
-      console.log(
-        "DEBUG: --------------------------------------------------------------------------------------------------------"
-      );
       console.log(itemWithDocuments.data.documents.length);
       console.log(Array.isArray(itemWithDocuments.data.documents));
       throw Error("Adding documents to a workflowitem failed :(");
     }
   }
 
-  // Updating an existing document shouldn't be allowed:
-  try {
-    await axios.post("/workflowitem.update", {
-      projectId: project.data.id,
-      subprojectId: subproject.data.id,
-      workflowitemId: workflowitem.data.id,
-      documents: [
-        {
-          id: documentId1,
-          base64:
-            "VGhhdCdzIG91ciBmaXJzdCBjb250cmFjdC4gSnVzdCBraWRkaW5nLCBJJ3ZlIGNoYW5nZWQgaXQhIG11YWhhaGE=",
-        },
-      ],
-    });
-    throw Error(
-      "Updated an existing document, but that isn't supposed to work :("
-    );
-  } catch (_err) {
-    // ignore error
-  }
-
```

Comment:
this is obsolete. Moreover, these business rules should not be here, it should be an API test